### PR TITLE
Add com.gitlab.tipp10.tipp10

### PR DESCRIPTION
**Pull request by a-andre**

### CODE REVIEW

**Comment by a-andre** [comment]
bot, build com.gitlab.tipp10.tipp10

File: com.gitlab.tipp10.tipp10.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+app-id: com.gitlab.tipp10.tipp10
+runtime: org.kde.Platform
+runtime-version: '5.14'
```
Comment (bilelmoussaoui) [inline com.gitlab.tipp10.tipp10.yml:3]: Have you tried to bump this to 5.15?

Comment (barthalion) [inline com.gitlab.tipp10.tipp10.yml:3]: There's 5.15 since today. Want to give it a try?

**Comment by nedrichards** [comment]
5.15 seems to work fine in manual testing:
> flatpak run --runtime=org.kde.Platform//5.15 com.gitlab.tipp10.tipp10

Other than that, seems to work well.

**Comment by a-andre** [comment]
bot, build com.gitlab.tipp10.tipp10

**Pull request closed without merging**